README.md

```diff
@@ -62,29 +64,26 @@ prowler <provider>
 ```
 ![Prowler CLI Execution](docs/img/short-display.png)
 
+## Prowler CLI Features
```

Comment:
This looks more like "Prowler Compliance Features" from the `prowler dashboard` than "Prowler CLI Features"